### PR TITLE
Update support matrix .

### PR DESCRIPTION
**Pull request by lipingxue**
Signed-off-by: Liping Xue <lipingx@vmware.com>

**What this PR does / why we need it**:
Update support matrix for Vanilla, WCP and GC.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```NONE

```

### CODE REVIEW

File: docs/guest.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ This document discusses the velero vSphere plugin installation process in a **Ta
 
 | Velero Plugin for vSphere Version | vSphere Version    | Kubernetes Version | vSphere CSI Driver Version | Velero Version | vSphere Plugin Deprecated | vSphere Plugin EOL Date      |
 |-----------------------------------|--------------------|--------------------|----------------------------|----------------|------------|---------------|
+| 1.5.1                             | 8.0U1               | 1.24          | Bundled with TKGS          | 1.10.2          | No         | N/A           |
+| 1.5.1                             | 8.0                | 1.24          | Bundled with TKGS          | 1.10.2          | No         | N/A           |
```
Comment (xing-yang) [inline docs/guest.md:20]: I think this should be 8.0c.  Let's ask @Ravi Kumar K P to verify.

File: docs/supervisor.md [hunk 2]
```diff
@@ -13,6 +13,8 @@
 
 | Velero Plugin for vSphere Version | vSphere Version     | Kubernetes Version                                                | vSphere CSI Driver Version | Velero Version | Velero vSphere Operator Version | Data Manager Version | vSphere Plugin Deprecated | vSphere Plugin EOL Date      |
 |-----------------------------------|---------------------|-------------------------------------------------------------------|----------------------------|----------------|---------------------------------|---------------|------------|---------------|
+| 1.5.1                             | 8.0U1                  | Bundled with vSphere (1.24)                                 | Bundled with vSphere       | 1.10.2          | 1.4.0                           | 1.2.0        | No         | N/A           |
+| 1.5.1                             | 8.0                  | Bundled with vSphere (1.24)                                 | Bundled with vSphere       | 1.10.2          | 1.4.0                           | 1.2.0        | No         | N/A           |
```
Comment (xing-yang) [inline docs/supervisor.md:17]: Same question here.

**Comment by xing-yang** [comment]
Can you also update the matrix for Vanilla?

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/820/

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/821/

File: docs/vanilla.md [hunk 3]
```diff
@@ -14,6 +14,7 @@
 
 | Velero Plugin for vSphere Version | vSphere Version        | Kubernetes Version | vSphere CSI Driver Version        | Velero Version | vSphere Plugin Deprecated | vSphere Plugin EOL Date      |
 |-----------------------------------|------------------------|--------------------|-----------------------------------|----------------|------------|---------------|
+ 1.5.1                             | 8.0U1                    | 1.26          | 3.0.1                             | 1.10.2          | No         | N/A           |
```
Comment (xing-yang) [inline docs/vanilla.md:17]: Both Velero versions 1.10.2 and 1.11.0 were tested for Vanila.

**Comment by xing-yang** [comment]
LGTM

**Pull request merged**